log.go

```diff
@@ -122,6 +122,15 @@ func SetupLoggers(root *build.RotatingLogWriter, interceptor signal.Interceptor)
 		SetSubLogger(root, l.subsystem, l.Logger)
 	}
 
+	// Initialize loggers from packages outside of `lnd` first. The
```

Comment:
:eyes:

There is no easy way to know what overwrites what beyond checking the code right?

Reply:
yea too bad nope. I ran into this issue while debugging a connection lost, and found logs started with `BTCN`, which was very confusing as I thought the neutrino was running😂